### DISCUSSION TITLE
PrimeNF blocks with tailwindcss?

### DISCUSSION THREAD

**Discussion by aaron-za-dev** [primefaces/community#1458] [PrimeNG Blocks] [answered]
Is it possible to use prime blocks with Tailwind? Are there any plans to support it?

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

This implementation is in progress right now, all block products will be compatible with tailwind soon.

**Comment by planitron** [comment]
Thanks, I had the same question and in the meantime ,

is there a way to use primeflex and tailwind or will it conflict?
